banjo/conf/schema.xml

```diff
@@ -219,7 +219,7 @@
    <field name="speed" type="text_general" indexed="true" stored="true"/>
    <field name="thickness" type="text_general" indexed="true" stored="true"/>
    <field name="brand" type="text_general" indexed="true" stored="true"/>
-   <field name="duration" type="int" indexed="true" stored="true"/>
+   <field name="duration_seconds" type="long indexed="true" stored="true"/>

```

Comment:
Duration field hasn't ben used so far.
 Based on our conversation I am adding another field duration_minutes, to be used as an additional field for banjo search.